test.js

```diff
@@ -13,53 +12,47 @@ describe('should find urls in fixture file', function () {
 		'http://github.com'
 	];
 
-	it('should get unique cleaned-up urls from a string', function () {
-		assert.deepEqual(getUrls(fs.readFileSync('fixture.txt', 'utf8')), expected);
-	});
+	t.same(fn(fs.readFileSync('fixture.txt', 'utf8')), expected);
```

Comment:
I would place the array directly in this line. Not sure what @sindresorhus likes though.